src/index.jsx

```diff
@@ -73,7 +73,7 @@ function Form() {
             onChange = {
                 (ev) => setHubUrl(ev.target.value)
             }
-            placeholder="https://myjupyterhub.org"
+            placeholder="https://example.edu"
```

Comment:
@ryanlovett
changed it to an example domain, however using edu, as to not confuse a professor who might otherwise say something like "https://datahub.berkeley.org" instead of ".edu"

Reply:
Using .edu is a great idea, thanks. I've confirmed that example.edu is a domain under the same IANA stewardship.